### PR TITLE
Enable GenericResponses for Flask-Security

### PR DESCRIPTION
**Pull request by quaxsze**
Fix https://github.com/etalab/data.gouv.fr/issues/1053

### CODE REVIEW

File: requirements/install.in [hunk 1]
```diff
@@ -22,7 +22,7 @@ Flask-Mail==0.9.1
 flask-mongoengine==0.9.5
 Flask-Navigation==0.2.0
 flask-restx==1.0.5
-Flask-Security-Too==4.1.5
+Flask-Security-Too==5.1.1
```
Comment (maudetes) [inline requirements/install.in:25]: 5.1.2 is out already ! :p
With a bit of additional doc on customizing that may be useful: https://github.com/Flask-Middleware/flask-security/pull/773/files

**Review by maudetes — COMMENTED** [review]
We're going to be cutting-edge! :partying_face:

File: udata/core/user/models.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ class Role(db.Document, RoleMixin):
     ADMIN = 'admin'
     name = db.StringField(max_length=80, unique=True)
     description = db.StringField(max_length=255)
+    permissions = db.ListField()
```
[the remark this answers is not in the document]
Comment (quaxsze) [reply udata/core/user/models.py:33]: Did not find any and the fact it adds a field does not seem to required any change outside flask security

File: udata/templates/security/email/welcome_existing.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{% extends 'security/email/base.txt' %}
+{% block body %}
+{{ _('Hello) }}
```
Comment (maudetes) [inline udata/templates/security/email/welcome_existing.txt:3]:
```suggestion
{{ _('Hello') }}
```

**Review by maudetes — COMMENTED** [review]
No message seems to be displayed when trying to login with wrong credentials

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,11 @@
     - Added `created_at` computed property in Dataset and Resource model to provide harvested date if present and internal otherwise
     - Added `last_modified` computed property in Dataset and Resource model to provide max date between internal and harvested date
 - Fix for PR [#2815](https://github.com/opendatateam/udata/pull/2815) [#2832](https://github.com/opendatateam/udata/pull/2832)
+- :warning: Flask-Security update to enable `GenericResponses` [#2826](https://github.com/opendatateam/udata/pull/2826):
```
Comment (maudetes) [inline CHANGELOG.md:18]: You should move this entry to the `Current` section

**Review by maudetes — APPROVED** [review]
Very glad that w'ere going to be up-to-date on Flask-Security :muscle:

**Pull request merged**